### PR TITLE
feat: pass orgrole when adding rs event

### PR DESCRIPTION
**Pull request by s4kh**
**What is this feature?**

Passes currently logged in user's role for EchoMeta. 

**Why do we need this feature?**

Differentiate user roles in tracked events

**Which issue(s) does this PR fix?**:

Fixes https://github.com/grafana/grafana-community-team/issues/230, https://github.com/grafana/grafana-community-team/issues/231

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.

### CODE REVIEW

File: packages/grafana-runtime/src/services/EchoSrv.ts [hunk 1]
```diff
@@ -24,17 +24,21 @@ export interface EchoMeta {
...
    */
   userId: number;
   /**
    * True when user is logged in into Grafana.
    */
   userSignedIn: boolean;
+  /**
+   * Current user's role
+   */
+  orgRole: string | '';
```
Comment (s4kh) [inline packages/grafana-runtime/src/services/EchoSrv.ts:41]: in [ContextSrv](https://github.com/grafana/grafana/blob/main/public/app/core/services/context_srv.ts#L36) the type is `orgRole: OrgRole | '';`

File: public/app/features/dashboard-scene/panel-edit/PanelVizTypePicker.tsx [hunk 2]
```diff
@@ -25,6 +26,13 @@ export function PanelVizTypePicker({ panel, data, onChange, onClose }: Props) {
   const styles = useStyles2(getStyles);
   const [searchQuery, setSearchQuery] = useState('');
 
+  const handleSearchChange = (value: string) => {
+    if (value) {
+      reportInteraction('dashboards_dspickermodal_clicked', { item: 'search', query: value });
```
Comment (s4kh) [inline public/app/features/dashboard-scene/panel-edit/PanelVizTypePicker.tsx:31]: Need to pass search result count, but filtering happens inside the [child component.](https://github.com/grafana/grafana/pull/94890/files#diff-dd6934a79d19977e13188e9cc17ddbf74896145f176e8de9faa938a6d87c853aR94)

File: go.sum [hunk 3]
```diff
@@ -1487,8 +1487,9 @@ github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf/go.mod h1:ybxpYRF
 github.com/alecthomas/units v0.0.0-20190717042225-c3de453c63f4/go.mod h1:ybxpYRFXyAe+OPACYpWeL0wqObRcbAqCMya13uyzqw0=
 github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d/go.mod h1:rBZYJk541a8SKzHPHnH3zbiI+7dagKZ0cgpgrD7Fyho=
 github.com/alecthomas/units v0.0.0-20211218093645-b94a6e3cc137/go.mod h1:OMCwj8VM1Kc9e19TLln2VL61YJF0x1XFtfdL4JdbSyE=
-github.com/alecthomas/units v0.0.0-20231202071711-9a357b53e9c9 h1:ez/4by2iGztzR4L0zgAOR8lTQK9VlyBVVd7G4omaOQs=
 github.com/alecthomas/units v0.0.0-20231202071711-9a357b53e9c9/go.mod h1:OMCwj8VM1Kc9e19TLln2VL61YJF0x1XFtfdL4JdbSyE=
+github.com/alecthomas/units v0.0.0-20240626203959-61d1e3462e30 h1:t3eaIm0rUkzbrIewtiFmMK5RXHej2XnoXNhxVsAYUfg=
```
Comment (s4kh) [inline go.sum:1491]: Ran due to https://github.com/grafana/grafana/actions/runs/11403494960/job/31730685400#step:5:113

File: packages/grafana-runtime/src/services/EchoSrv.ts (hunk 1 above)
Comment (xnyo) [reply packages/grafana-runtime/src/services/EchoSrv.ts:41]: I think `OrgRole | ''` is more appropriate :)

**Review by xnyo — APPROVED** [review]
LGTM!

**Pull request merged**